STYLE_GUIDE.md

```diff
@@ -154,6 +154,27 @@ Prefer:
 
 ## JavaScript
 
+### Variable naming
+
+- Prefer camelCase over snake_case, except in JSON.
```

Comment:
Maybe highlight as code like: `camelCaseName` over `snake_case_name`

Reply:
idk, I feel like that gives the impression it's referring to a specific code thing?